designs/virtualization/4.5/Scheduling/Affinity/Affinity.md

```diff
@@ -0,0 +1,65 @@
+---
+parent: Virtualization
+version: 4.5
+---
+
+# Affinity
+
+Introducing Affinity as it appears in-page and the Edit modal.
+
+## in-page placement
+
+![Affinity field in the Scheduling section ](img/Affinity-0-0.jpg)
+The affinity field is placed in the Workload details page > Details tab > Scheduling section > Policies columns.
+
+![Affinity field in the Scheduling section ](img/Affinity-0-1.jpg)
+The affinity field counts the number of rules applied.
+
+## Affinity rules list
+
+![Affinity all rules modal ](img/Affinity-1-0.jpg)
```

Comment:
Should this be "View affinity rules" ? And I'm not sure we use "Done" as the primary action for modals anywhere else. If users can delete rules from here, should the actions actually be save and cancel?

Reply:
Sounds right, I'll update it to Save/Cancel